test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -217,7 +217,7 @@ public void NgramWorkout()
             var est = new WordTokenizingEstimator(Env, "text", "text")
                 .Append(new ValueToKeyMappingEstimator(Env, "text", "terms"))
                 .Append(new NgramExtractingEstimator(Env, "terms", "ngrams"))
-                .Append(new NgramHashEstimator(Env, "terms", "ngramshash"));
+                .Append(new NgramHashingEstimator(Env, "terms", "ngramshash"));
```

Comment:
Is there any benefit to doing a workout test on just the ngram hash estimator? For example, will the invalid data get caught by another transform? 

Also what about more tests around the estimator class itself? Things like Saving (old and new as it looks like the version was revved), checking the output schema passing in bad values on construction, etc.

Reply:
Would be good to see that you can load the old format

---
In reply to: [238489293](https://github.com/dotnet/machinelearning/pull/1811#discussion_r238489293) [](ancestors = 238489293)

Reply:
I can't load old format, I broke backward compatibility with this changes.


---
In reply to: [238835998](https://github.com/dotnet/machinelearning/pull/1811#discussion_r238835998) [](ancestors = 238835998,238489293)

Reply:
To @singlis@microsoft.com  we are doing workout test here already.

---
In reply to: [238839172](https://github.com/dotnet/machinelearning/pull/1811#discussion_r238839172) [](ancestors = 238839172,238835998,238489293)

Reply:
Is it fine to break backward compatibility?

---
In reply to: [238839333](https://github.com/dotnet/machinelearning/pull/1811#discussion_r238839333) [](ancestors = 238839333,238839172,238835998,238489293)